extras/userdoc/md/documentation/installation.md

```diff
@@ -1,87 +1,82 @@
+directory. For using NEST on Microsoft Windows, see [below](#windows).
 
 Following are the basic steps to compile and install NEST from source code:
 
 1.  [Download NEST](download.md)
-
-2.  Unpack the tarball: `tar -xzvf nest-x.y.z.tar.gz`
-
-3.  Create a build directory: `mkdir nest-x.y.z-build`
-
-4.  Change to the build directory: `cd nest-x.y.z-build`
-
-5.  Configure NEST: `../nest-x.y.z/configure` with appropriate
-     configuration options (**prefix**, etc.)
-
-6.  Compile by running `make`
-
-7.  Install by running `make install`
-
-8.  See the [Getting started](getting-started.md) pages to
+1.  Unpack the tarball: `tar -xzvf nest-x.y.z.tar.gz`
+1.  Create a build directory: `mkdir nest-x.y.z-build`
+1.  Change to the build directory: `cd nest-x.y.z-build`
+1.  Configure NEST: `cmake -DCMAKE_INSTALL_PREFIX:PATH=</install/path> </path/to/NEST/src>` with
+additional `cmake` options as needed (see `INSTALL` file)
+1.  Compile by running `make`
+1.  Install by running `make install`
+1.  Run tests by running `make installcheck`
+1.  See the [Getting started](getting-started.md) pages to
     find out how to get going with NEST
 
-Please see the sections on [minimal](installation.md#minimal-configuration) and
-[standard configuration](installation.md#standard-configuration)
+Please see the sections on [minimal](#minimal-configuration) and
+[standard configuration](#standard-configuration)
 below for details.
 
 ## What gets installed where
 
 By default, everything will be installed to the subdirectories
-`$PREFIX/{bin,lib,share}`, where `$PREFIX=$HOME/opt/nest`:
+`/install/path/{bin,lib,share}`, where `/install/path` is the install path given to `cmake`:
 
 Executables   
-`$PREFIX/bin`
+`/install/path/bin`
 
-Extensions   
-`$PREFIX/lib/nest`
+Dynamic libraries   
+`/install/path/lib/`
 
 SLI libraries   
-`$PREFIX/share/nest/sli`
+`/install/path/share/nest/sli`
 
 Documentation   
-`$PREFIX/share/doc/nest`
+`/install/path/share/doc/nest`
 
 Examples   
-`$PREFIX/share/doc/nest/examples`
+`/install/path/share/doc/nest/examples`
 
 PyNEST   
-`$PREFIX/lib/pythonX.Y/site-packages/nest`
+`/install/path/lib/pythonX.Y/site-packages/nest`
 
 PyNEST examples   
-`$PREFIX/share/doc/nest/examples/pynest`
+`/install/path/share/doc/nest/examples/pynest`
 
 Extras   
-`$PREFIX/share/nest/extras/emacs/sli`
+`/install/path/share/nest/extras/`
 
-You can supply a custom prefix using the `--prefix=$PREFIX` switch for
-`./configure`. In this case, you have to add the installation directory to your
+If you want to run the `nest` executable or use the `nest` Python module without
+providing explicit paths, you have to add the installation directory to your
 search paths. For example, if you are using bash:
 
-    export PATH=$PATH:$PREFIX/bin
-    export PYTHONPATH=$PREFIX/lib/pythonX.Y/site-packages:$PYTHONPATH
+    export PATH=$PATH:/install/path/bin
+    export PYTHONPATH=/install/path/lib/pythonX.Y/site-packages:$PYTHONPATH
 
```

Comment:
Add the following

The script `/install/path/bin/nest_vars.sh` can be [sourced](http://www.tldp.org/HOWTO/Bash-Prompt-HOWTO/x237.html) in `.bashrc` and will set these paths for you. This also allows to switch between NEST installations in a convenient manner.